### ISSUE TITLE
Using pictureboxes for buttons really screws with accessibility

### ISSUE THREAD

**Issue by zersiax** [hama3254/Crunchyroll-Downloader-v3.0#322]
I was trying to make this app more accessible for screenreader and keyboard-only users, but you seem to have used the pictureBox control for almost every button. PictureBoxes cannot receive tab index, and are really meant for showing off pictures only. You CAN attach click events to them, true, but this really is not a good pattern to follow.
Please consider using button controls with the style set to flat in order to essentially get the same look and much better accessibility.

**Comment by hama3254** [comment]
i can change that for sure, i just tried it with the first two button on the main form and i get it to look identical and with the `LostFocus` and `GotFocus` events i can add the hover effect while it is selected with the tab index.
These changes might take some time since i just started to switch the embedded browser from `GeckoFX` to `CefSharp` which is not the simplest task.
![grafik](https://user-images.githubusercontent.com/52826614/126043476-73e92a54-9ce8-4fa8-8e1e-52d0a59d2ef6.png)

**Comment by hama3254** [comment]
i released an improved version for more accessible as ` Crunchyroll-Downloader-Chromium-U1` feedback would be nice
https://github.com/hama3254/Crunchyroll-Downloader-v3.0/releases/tag/v3.x
